qualtran/bloqs/basic_gates/rotation_test.py

```diff
@@ -61,3 +61,24 @@ def test_as_cirq_op():
     op, _ = bloq.as_cirq_op(SimpleQubitManager(), **quregs)
     circuit = cirq.Circuit(op)
     assert circuit == cirq.Circuit(cirq.Rz(rads=bloq.angle).on(cirq.NamedQubit("q")))
+    bloq = XPowGate(exponent=1 / 5, global_shift=-0.5)
+    quregs = bloq.signature.get_cirq_quregs()
+    op, _ = bloq.as_cirq_op(SimpleQubitManager(), **quregs)
+    circuit = cirq.Circuit(op)
+    assert circuit == cirq.Circuit(
+        cirq.XPowGate(exponent=1 / 5, global_shift=-0.5).on(cirq.NamedQubit("q"))
+    )
```

Comment:
Is it worth testing the tensor contraction route to a unitary, or is this covered enough already?

Reply:
I've refactored the code so that these gates now derive from `CirqGateAsBloqBase`  abstract class. This has the added benefit that all the convenience methods (like `add_my_tensors` etc.) get auto populated and it gives a nicer way to wrap Cirq-Core gates that we need frequently in Qualtran as Bloqs.